benchmarking/runs/CONVENTION.md

```diff
@@ -2,4 +2,6 @@
 This directory contains simulation runs.
 Try to be descriptive when naming new files here.
 
+- Every file should contain a main function
```

Comment:
Every file should have a "main clause" that allows them to be run from the terminal:
```python
if __name__ == "__main__":
    func()
```


My suggestion ^

Reply:
I see a few projects use this pattern, not so different from Opey's suggestion
```python
def main():
    func()
if __name__ == "__main__": 
    main()
```

Reply:
I kinda like the functions having names though, in case we make like "batch run" files that call many of them in sequence

Reply:
Oh ohhh I see what you're saying now my bad

Reply:
Idk if this is needed either but I kinda see why people do this? Let's leave as is for now